dev/docker/ballista.dockerfile

```diff
@@ -42,9 +42,9 @@ COPY datafusion-cli ./datafusion-cli/
 COPY datafusion-examples ./datafusion-examples/
 RUN cargo chef prepare --recipe-path recipe.json
 
-FROM base as cacher
-COPY --from=planner /tmp/ballista/recipe.json recipe.json
-RUN cargo chef cook $RELEASE_FLAG --recipe-path recipe.json
+# FROM base as cacher
+# COPY --from=planner /tmp/ballista/recipe.json recipe.json
+# RUN cargo chef cook $RELEASE_FLAG --recipe-path recipe.json
```

Comment:
I could use some help figuring out how to keep this working. See the issue for more details.